addons/addon-base-ui/packages/base-ui/src/models/authentication/Authentication.js

```diff
@@ -175,8 +175,9 @@ const Authentication = types
       }
     },
     async logout({ autoLogout = false } = {}) {
+      const idToken = storage.getItem(localStorageKeys.appIdToken);
       self.cleanup();
-      return self.selectedAuthenticationProvider.logout({ autoLogout });
+      return self.selectedAuthenticationProvider.logout(idToken, { autoLogout });
```

Comment:
how about logging out when using Cognito user pool? what's the expected value of `idToken`?

Reply:
Cognito did not use idToken, the signOutUri is: <baseAuthUri>/logout?client_id=<clientId>&response_type=code&redirect_uri=<websiteUrl>